ddtrace/appsec/_remoteconfiguration.py

```diff
@@ -26,9 +33,46 @@ def enable_appsec_rc():
 
     if _appsec_rc_features_is_enabled():
         from ddtrace.internal.remoteconfig import RemoteConfig
+        from ddtrace.internal.remoteconfig.constants import ASM_DATA_PRODUCT
         from ddtrace.internal.remoteconfig.constants import ASM_FEATURES_PRODUCT
 
         RemoteConfig.register(ASM_FEATURES_PRODUCT, appsec_rc_reload_features(tracer))
+        RemoteConfig.register(ASM_DATA_PRODUCT, appsec_rc_reload_features(tracer))
+
+
+def _appsec_rules_data(tracer, features):
+    # type: (Tracer, Union[Literal[False], Mapping[str, Any]]) -> None
```

Comment:
same question about the `False`, can we just use an empty map?

Reply:
@avara1986 replied on the other PR, sorry. Right it's like a tri-state. Weird and we definitely want to change it in the future.